app/src/main/java/openfoodfacts/github/scrachx/openfood/views/ProductImageManagementActivity.java

```diff
@@ -125,6 +133,37 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
         onRefresh(false);
     }
 
+    private void ShowCaseIntro(String title, String content, int viewId, final int type) {
```

Comment:
Follow methods naming style
```suggestion
    private void startShowcase(String title, String content, int viewId, final int type) {
```